spaghetto/core/src/Config.purs

```diff
@@ -94,13 +94,17 @@ type PublishConfig =
   { version :: Version
   , license :: License
   , location :: Maybe Location
+  , include :: Maybe (Array FilePath)
+  , exclude :: Maybe (Array FilePath)
```

Comment:
These are really globs, right? Should we newtype and sanitize this the way it’s done in the registry?

https://github.com/purescript/registry-dev/blob/6c1209c9eaad10269a2e33445f9beb61b7ac0d12/lib/src/Internal/Path.purs#L29

Reply:
This part of the PR is about specifying things in the config file. I am not sure we'd want to parse/validate things here already, all this stuff is mostly about reading things in.

Even in the registry the `sanitizePath` function that you link is only used at glob-matching time, inside `Registry.Foreign.FastGlob.match'`.
We use that same function to match globs here, so if I understand correctly we're already getting the sanitization for free.

Reply:
Oh so it's impossible to add sources from a sibling project in the same project? I guess we can always relax that if the need ever arises.

Reply:
It should be impossible to add sources from a sibling project as far as the registry (publishing) is concerned, though I suppose it’s fine for Spago

Reply:
I don't think it should be fine for Spago either, the proper mechanism for adding sibling projects is through a package dependency.